lib/router/schema_definitions.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Last modified: 11/08/2016",
+  "description": "Last modified: 12/12/2016",
```

Comment:
nit: any way to make this automatic? seems like version IDs and commits for date/history would be nicer